lib/indexer/records.js

```diff
@@ -0,0 +1,152 @@
+/*!
+ * records.js - indexer records
+ * Copyright (c) 2018, the bcoin developers (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+/**
+ * @module lib/records
+ */
+
+const bio = require('bufio');
+const util = require('../utils/util');
+const consensus = require('../protocol/consensus');
+
+/**
+ * Block Meta
+ * @alias module:indexer.BlockMeta
```

Comment:
I think this class could use some assertions based on the types and/or some tests